DiscordChatExporter.Core/Exporting/MessageExporter.cs

```diff
@@ -30,6 +30,11 @@ private async ValueTask ResetWriterAsync(CancellationToken cancellationToken = d
         }
     }
 
+    // Unless explicitly called, file will be created before writing first message
+    public async ValueTask EnsureFileIsCreated(CancellationToken cancellationToken = default) {
```

Comment:
It might be better to call this implicitly during disposal instead.

Reply:
Done in https://github.com/Tyrrrz/DiscordChatExporter/pull/1349/commits/27a7bdd8702e41ad95566a3b6797c3a52f74924a